admin_guide/router.adoc

```diff
@@ -10,45 +10,47 @@
 toc::[]
 
 == Overview
-Once you have a running link:../architecture/core_objects/routing.html[Router] you are ready
-to monitor it.  Depending on the underlying router implementation this may be accomplished
-multiple ways.  This document will discuss the HAProxy template router and the different
-components that can be checked to ensure its health.
+Depending on the underlying implementation, you can monitor a running
+link:../architecture/core_objects/routing.html[router] in multiple ways. This
+topic discusses the HAProxy template router, and the components to check to
+ensure its health.
 
 == Viewing Statistics
-The HAProxy router exposes a web listener for the HAProxy statistics.  You can enter the public
-IP address of the router with the correctly configured port to view the statistics page.  You 
-will be prompted for the admin password.  This password and port were configured during the
-router installation.
+The HAProxy router exposes a web listener for the HAProxy statistics. Enter the
+router's public IP address and the correctly configured port (1936 by default)
+to view the statistics page. You will be prompted for the admin password. This
```

Comment:
True. Changed to:

"Enter the router's public IP address and the correctly configured port (1936 by default) to view the statistics page and enter the admin password when prompted."